Automattic/jetpack
# Donation block: Payment modal not displayed on iOS devices

Fixes https://github.com/Automattic/wp-calypso/issues/55279

#### Changes proposed in this Pull Request:

* Removed a CSS style that was causing the modal to no be shown properly on iOS devices.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable? Not applicable

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






##### Atomic sites testing
* Go to an Atomic site and use Jetpack beta to use this branch.
* Make sure your proxy is disabled.
* Navigate to a site with a donations block. (You can use [this site/post ](https://atomic20220811pool.wpcomstaging.com/2022/08/16/foooing/)for example)
* The modal should be displayed correctly.
* Repeat these steps on iOS devices, desktop, and as many width combinations as possible.

##### Simple sites testing
* You should also test that it works properly on simple sites. Use the steps above (excluding going to an Atomic site)

##### Test evidence

https://user-images.githubusercontent.com/1989914/185604214-1bc693fd-6c00-4a86-bd9c-d9a6e2a03766.mp4
